packages/medusa-payment-stripe/src/api/routes/hooks/stripe.js

```diff
@@ -1,3 +1,5 @@
+import { PostgresError } from "@medusajs/medusa/src/utils"
```

Comment:
comment: Should this be exported from index at root so we can import it like so: `import {} from @medusajs/medusa`?

Reply:
non-blocking

Reply:
yes defiantly , I was not sure about it but if you agree then yes